specification/assets/gherkin/evaluation_v2.feature

```diff
@@ -0,0 +1,603 @@
+  # Note: Test data setup ensures the flag returns wrong type, triggering type mismatch
+  # Implicitly tests: 1.4.10 (client never throws exceptions - returns default instead)
+    @error-handling @spec-1.3.4 @spec-1.4.10
+    Scenario Outline: Type mismatch error
+        Given a <requested_type>-flag with key "<key>" and a default value "<default>"
+        When the flag was evaluated with details
+        Then the resolved details value should be "<default>"
+        And the reason should be "ERROR"
+        And the error-code should be "TYPE_MISMATCH"
+
+        @booleans
+        Examples: Boolean evaluations
+            | key          | requested_type | default |
+            | boolean-flag | String         | bye     |
+
+        @strings
+        Examples: String evaluations
+            | key         | requested_type | default |
+            | string-flag | Boolean        | false   |
+
+        @numbers
+        Examples: Number evaluations
+            | key         | requested_type | default |
+            | string-flag | Integer        | 1       |
+
+        @objects
+        Examples: Object evaluations
+            | key         | requested_type | default |
+            | string-flag | Object         | {}      |
+
+  # Spec 1.7.6: Testing PROVIDER_NOT_READY error when provider isn't initialized
+  # Testing: client must short-circuit and return error when provider not ready
+  # Implicitly tests: 1.4.10 (client never throws exceptions - returns default instead)
+    @provider-status @spec-1.7.6 @spec-1.4.10
+    Scenario Outline: Provider not ready error
+        Given a not ready provider
+        And a <type>-flag with key "<key>" and a default value "<default>"
+        When the flag was evaluated with details
+        Then the resolved details value should be "<default>"
+        And the reason should be "ERROR"
+        And the error-code should be "PROVIDER_NOT_READY"
+
+        @booleans
+        Examples: Boolean evaluations
+            | key          | type    | default |
+            | boolean-flag | Boolean | false   |
+
+        @strings
+        Examples: String evaluations
+            | key         | type   | default |
+            | string-flag | String | bye     |
+
+        @numbers
+        Examples: Number evaluations
+            | key          | type    | default |
+            | integer-flag | Integer | 1       |
+            | float-flag   | Float   | 0.1     |
+
+        @objects
+        Examples: Object evaluations
+            | key         | type   | default    |
+            | object-flag | Object | {\"a\": 1} |
+
+  # Spec 1.7.7: Testing PROVIDER_FATAL error when provider is in fatal state
+  # Testing: client must short-circuit and return error when provider is fatal
+  # Implicitly tests: 1.4.10 (client never throws exceptions - returns default instead), 1.7.5 (FATAL status)
+    @provider-status @spec-1.7.7 @spec-1.4.10 @spec-1.7.5
```

Comment:
```suggestion
    @provider-status @provider-status-fatal @spec-1.7.7 @spec-1.4.10 @spec-1.7.5
```

Similar to https://github.com/open-feature/spec/pull/336/files#r2315433976, I think... but use your judgement.